main.go

```diff
@@ -94,7 +92,14 @@ func main() {
 	templateVars["KEYCLOAK_OSO_ENDPOINT"] = keycloakConfig.CustomBrokerTokenURL("openshift-v3")
 	templateVars["KEYCLOAK_GITHUB_ENDPOINT"] = fmt.Sprintf("%s%s?for=https://github.com", config.GetAuthURL(), authclient.RetrieveTokenPath())
 
-	publicKeys, err := token.GetPublicKeys(context.Background(), config.GetAuthURL())
+	authService, err := auth.NewAuthService(config)
+	if err != nil {
+		log.Panic(nil, map[string]interface{}{
+			"err": err,
+		}, "failed to fetch service account token")
```

Comment:
something wrong with error msg ;-)

Reply:
??? I'm probably blind...

Reply:
in case `NewAuthService()` return error, we should logging relevant error_msg and not "failed to fetch service account token". :-)

Reply:
changed in 65ea36f